CONTRIBUTING.md

```diff
@@ -40,11 +40,12 @@ As you work...
   using [True](http://oddbird.net/true) --
   in the `test` directory,
   and make sure all tests are passing with `yarn test`
-- Add your changes to the [changelog](CHANGELOG.md)
+- Add your changes to the [changelog][]
 - Update or add inline documentation
   using [SassDoc][sassdoc] as appropriate,
   and compile the docs with `yarn docs`
 
+[changelog]: https://github.com/oddbird/accoutrement/blob/master/CHANGELOG.md
```

Comment:
Better to link them in the docs rather than on a particular gihub branch (especially one we'll likely change soon)?

Reply:
Up to you.

Reply:
let's link to docs

Reply:
Ok -- the only thing that seems odd about that is this is in the Contributing doc, referencing a file to edit (not just read). 🤷

Reply:
oh, that's true. Ok, we can update the branch once we change it…